.github/workflows/run-tests.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Run tests with pytest
         run: |
           pytest -v --durations=0 --cov=osbot_utils tests
-          coveralls
+          coveralls --service=github
```

Comment:
Coverall was failing for me, I added this as per the recommendation in https://github.com/TheKevJames/coveralls-python/issues/251

Reply:
This shouldn't be needed (none of the other services have this)

Did it work when you made this change?